src/test/resources/features/AddProductToCart.feature

```diff
@@ -5,6 +5,53 @@ Feature: Add product to Shopping cart
     Given user is on Home page
 
   Scenario: Validate access to subcategory
-    When User clicks on 'electronics' category from category bar
-    And User clicks on 'cell-phones' subcategory
-    Then 'Cell phones' subcategory page is displayed
\ No newline at end of file
+    When user clicks on 'electronics' category from category bar
```

Comment:
you can put the first two steps in the background

Reply:
Done.
Thank You!